src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Models/NotificationDataV2.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Models
```

Comment:
`models` namespace is not a newly added namespace.
Are you suggesting I move the `NotificationDataV2` class out of the `models` namespace or get rid of the namespace altogether?

Reply:
At the least I'd suggest not having this in that namespace. I'd say removing the models namespace altogether is a good idea, not sure if it will pollute this PR though.

Reply:
Done!